pkg/arvo/sur/btc-node-hook.hoon

```diff
@@ -0,0 +1,2489 @@
+                include-empty=(unit ?)
+              ::
+                include-watch-only=(unit ?)
+              ::
+        ==  ==
+        ::  %lists-in-ceblock: Get all transactions in blocks since block
+        ::  [blockhash], or all transactions if omitted.
+        ::
+        $:  %lists-in-ceblock
+            %-  unit
+            $:  blockhash=(unit blockhash)
+              ::
+                target-confirmations=(unit @ud)
+              ::
+                include-watch-only=(unit ?)
+              ::
+                include-removed=(unit ?)
+        ==  ==
+        ::  %list-transactions: If a label name is provided, this will return
+        ::  only incoming transactions paying to addresses with the specified
+        ::  label.
+        ::
+        $:  %list-transactions
+            %-  unit
+            $:  label=(unit @t)
+              ::
+                count=(unit @ud)
+              ::
+                skip=(unit @ud)
+              ::
+                include-watch-only=(unit ?)
+        ==  ==
+        ::  %list-unspent: Returns array of unspent transaction outputs
+        ::  (with between minconf and maxconf (inclusive) confirmations.
+        ::
+        $:  %list-unspent
+            %-  unit
+            $:  minconf=(unit @ud)
+              ::
+                maxconf=(unit @ud)
+              ::
+                addresses=(unit (list address))
+              ::
+                include-unsafe=(unit ?)
+              ::
+                $=  query-options
+                %-  unit
+                $:  minimum-amount=(unit @ud)
+                  ::
+                    maximum-amount=(unit @ud)
+                  ::
+                    maximum-count=(unit @ud)
+                  ::
+                    minimum-sum-amount=(unit @ud)
+        ==  ==  ==
+        ::  %list-wallet-dir  Returns a list of wallets in the wallet directory.
+        ::
+        [%list-wallet-dir ~]
+        ::  %list-wallets  Returns a list of currently loaded wallets.
+        ::  (For full information on the wallet, use "getwalletinfo"
+        ::
+        [%list-wallets ~]
+        ::  %load-wallet  Loads a wallet from a wallet file or directory.
+        ::
+        [%load-wallet filename=@t]
+        ::  %lock-unspent: Updates list of temporarily unspendable outputs.
+        ::
+        $:  %lock-unspent
+            unlock=?
+          ::
+            transactions=(unit (list [txid=@ux vout=@ud]))
+        ==
+        ::  %remove-pruned-funds  Deletes the specified transaction from the
+        ::  wallet.
+        ::
+        [%remove-pruned-funds txid=@ux]
+        ::  %rescan-blockchain  Rescan the local blockchain for wallet related
+        ::  transactions.
+        ::
+        [%rescan-blockchain start-height=(unit @ud) stop-height=(unit @ud)]
+        ::  %send-many: Send multiple times.
+        ::  Amounts are double-precision floating point numbers.
+        ::
+        $:  %send-many
+            dummy=%''
```

Comment:
I think you want `%$`, which is equivalent, but more-correct notation for "empty constant".